PowerEditor/src/WinControls/FunctionList/functionListPanel.cpp

```diff
@@ -350,7 +350,7 @@ void FunctionListPanel::reload()
 	
 	if (root)
 	{
-		Buffer* currentBuf = (*_ppEditView)->getCurrentBuffer();

```

Comment:
**Buffer* currentBuf** is initialized in Line 324 and remains unchanged. If you wanted, this entire line could be dropped.